wg-manifests/charter.md

```diff
@@ -0,0 +1,100 @@
+# WG Manifests Charter
+
+This charter adheres to the conventions, roles and organization management
+outlined in [wg-governance].
+
+## Scope
+
+- Provide a catalog (centralized repository) of Kubeflow application manifests.
+- Provide a catalog of third-party apps for common services.
+- Provide documentation to help users install a set, or all of the included
+  apps.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Maintain a set of manifests that will allow users to install Kubeflow apps and
```

Comment:
Are you actually maintaining the manifests? I thought it would be the application owners that would be maintaining the manifests?

My expectation is that you would be maintaining tooling to automate the copying of the manifests.

Reply:
Will reword